packages/nuxt/src/app/nuxt.ts

```diff
@@ -224,6 +225,7 @@ export function createNuxtApp (options: CreateOptions) {
     payload: reactive({
       data: {},
```

Comment:
the payload (when payload extraction is enabled) will only include `data`.

if that's the desired behaviour (that is, the same as `useState`) then we should probably document it. we could also make it clearer by storing it _within_ state as `state.once`

Reply:
I don't think it makes sense regarding `state.once` since you can also use this to log things and they may not be related to state management.

It is fine being outside data anyway since only `useFetch` or `useAsyncData` should be extracted for payload extraction.

PS: I tested an example with `nuxt generate` and it works like intended.